<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>set</class>
 <widget class="QMainWindow" name="set">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnPreMusic">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>84</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlay">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>84</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPause">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>84</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNextMusic">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>84</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddMusic">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>84</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVolume">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>84</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QListView" name="lwMusicList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>381</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="hSliderPlayProgress">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vSliderVolume">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>18</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>526</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
